### PR TITLE
Basic rich text support for ballot measures

### PR DESCRIPTION
**Pull request by jonahkagan**
## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/hmpb/layout/src/document_svg.tsx [hunk 1]
```diff
@@ -48,6 +50,50 @@ export function SvgBubble({
   return <SvgRectangle {...props} />;
 }
 
+function splitIntoHtmlSpans(html: string): React.ReactNode[] {
+  // Remove any <html> and </html> tags in the line
+  // eslint-disable-next-line no-param-reassign
+  html = html.replace(/<\/?html>/, '');
+
+  // DOMParser breaks if there's a closing tag with no opening tag, so fix that
+  // It doesn't mind unclosed tags, so we don't need to fix those
```
Comment (eventualbuddha) [inline libs/hmpb/layout/src/document_svg.tsx:59]: Why would we encounter closing tags with no opening tags. Is this to do with splitting into lines?

Comment (jonahkagan) [reply libs/hmpb/layout/src/document_svg.tsx:59]: Yep, lines get split higher up the stack

Comment (eventualbuddha) [reply libs/hmpb/layout/src/document_svg.tsx:59]: I think it'd be easier for me to understand the intent of this code if there were some tests.

**Review by eventualbuddha — APPROVED** [review]
LGTM. Thanks for adding the tests. I made a small tweak in #4577, but maybe you want to allow that? Your call.

**Pull request merged**